online-server/src/main/java/eu/itesla_project/online/server/message/ConnectionMessage.java

```diff
@@ -29,9 +31,13 @@ protected String getType() {
 
     @Override
     public String toJson() {
-        Gson gson = new Gson();
-        return gson.toJson(this);
-
+        ObjectMapper json = new ObjectMapper();
+        json.setVisibility(PropertyAccessor.FIELD, Visibility.ANY);
+        try {
+            return json.writeValueAsString(this);
+        } catch (Exception e) {
+            return "";
```

Comment:
I'm not sure that it's a good idea to return an empty JSON here. You should probably throw a RuntimeException

Reply:
ok